examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -164,6 +164,36 @@ def unflatten(theta: array) -> Dict[Handler, array]:
     return theta, unflatten
 
 
+def run_validation_edges(validation: Dataset, params, systems, epoch):
+    if len(validation) <= 0:
+        return
+    val_loss = np.zeros(len(validation))
+    for i, rfe in enumerate(validation.data):
+        if getattr(rfe, "complex_path", None) is not None:
```

Comment:
This looks a little janky

Reply:
It is, and can probably go away. Had it as I had older transformations that were generated that didn't have an associated path so I would default to the one provided by the script. The same logic happens for the training code